examples/online/index.js

```diff
@@ -27,9 +27,13 @@ var app = express();
 // activity tracking, in this case using
 // the UA string, you would use req.user.id etc
 
-app.use(function(req, res, next){
-  // fire-and-forget
-  online.add(req.headers['user-agent']);
+app.use(async (req, res, next) => {
+  try {
+    // fire-and-forget
```

Comment:
It is not fire and forget with an `await` and `try/catch`. The original design of this was because they didn't want to fail the request if the write failed and still fall back to the response handling.

That said, as an example goes I would prefer we just handle this type of error more robustly to show how. So I would suggest you change the `catch` to not pass the error to next, just `console.error(err)` it.